roles/prep_shared_data/defaults/main.yml

```
@@ -23,8 +23,8 @@ grpc4bmi_images:
       apptainer: ewatercycle-hype-grpc4bmi_feb2021.sif
     - docker: ghcr.io/ewatercycle/leakybucket-grpc4bmi:v0.0.1
       apptainer: ewatercycle-leakybucket-grpc4bmi_v0.0.1.sif
-    - docker: ghcr.io/daafip/hbv-bmi-grpc4bmi:v1.5.0
-      apptainer: daafip-hbv-bmi-grpc4bmi_v1.5.0.sif
+    - docker: ghcr.io/ewatercycle/hbv-bmi-grpc4bmi:v1.5.2
+      apptainer: ewatercycle-hbv-bmi-grpc4bmi_v1.5.2.sif
```

On https://github.com/eWaterCycle/HBV-bmi/pkgs/container/hbv-bmi-grpc4bmi I only see latest version

```suggestion
    - docker: ghcr.io/ewatercycle/hbv-bmi-grpc4bmi:latest
      apptainer: ewatercycle-hbv-bmi-grpc4bmi_v1.5.2.sif
```

or

```shell
docker tag ghcr.io/ewatercycle/hbv-bmi-grpc4bmi:latest ghcr.io/ewatercycle/hbv-bmi-grpc4bmi:v.1.5.2
docker push ghcr.io/ewatercycle/hbv-bmi-grpc4bmi:v.1.5.2
```